content/applications/finance/fiscal_localizations/belgium.rst

```diff
@@ -377,70 +366,49 @@ Once the connection is established, Odoo can be synchronized with CodaBox.
          :menuselection:`Accounting --> Configuration --> Accounting: CodaBox`.
 
 .. note::
-    The connection between Odoo and CodaBox can be revoked by using the
-    :guilabel:`Revoke` button in the CodaBox settings, which will revoke
-    the link on Odoo's side only.
-
-    If you have multiple connections, you will still be able to use the
-    Access Token of one of them to easily reconnect Odoo and CodaBox for
-    another client.
-
-    However, if you have only one (remaining) connection, you will not be able
-    to use its Access Token to create new connections. In that case, you will
-    have to revoke the connection from the CodaBox platform too,
-    and then recreate a new one following the steps in the
-    :ref:`first connection <belgium/codabox-configuration-connection>` section.
+    The connection between Odoo and CodaBox can be revoked by going to
+    :menuselection:`Accounting --> Configuration --> Accounting: CodaBox --> Manage Connection`
+    and clicking on the :guilabel:`Revoke` button.
 
 Potential issues
 ~~~~~~~~~~~~~~~~
 
 CodaBox is not configured. Please check your configuration.
   Either the Company VAT of the Accounting Firm VAT is not set.
 
-It seems that the fiduciary VAT number you provided is not valid. Please check your configuration.
-  Either the Company VAT of the Accounting Firm VAT is in the wrong format.
+No connection exists with these fiduciary and company VAT numbers. Please check your configuration.
+  This can happen when checking the connection status, and the pair (Fiduciary VAT, Company VAT)
+  is not registered yet. This may happen if you have changed the company VAT after the connection
```

Comment:
This can happen when checking the connection status, and the pair (Fiduciary VAT, Company VAT) still needs to be registered.